packages/ember-routing-views/lib/views/link.js

```diff
@@ -320,7 +320,7 @@ var LinkComponent = EmberComponent.extend({
     }
 
     var routing = get(this, '_routing');
-    var targetRouteName = get(this, 'targetRouteName');
+    var targetRouteName = this._handleOnlyQueryParamsSupplied(get(this, 'targetRouteName'));
```

Comment:
This looks good to me but in hindsight, but in hindsight we should probably find a better name for this method... it doesn't seem to "handle" anything but rather compute the target route name? @juggy, you're the original author, would you be ok with renaming this method or is there something I'm missing?